docs/packages/lucide.md

```diff
@@ -98,8 +102,8 @@ createIcons({
   icons: {
     Menu,
     ArrowRight,
-    Globe,
-  },
+    Globe
```

Comment:
Wow, great work, that's a _huge_ list of improvements and I think most of them are definitely no-brainers e.g. splitting alternate scripts is definitely the way to go.

The only thing I'm not quite sure about is — you guessed it — the lack of trailing commas, especially in light of Lucide scripts themselves using a code style that _does_ use trailing commas.
I'm personally persuasible, although I perhaps do lean more towards having trailing commas if only because of cleaner diffs. What do you think, @ericfennis?

Reply:
I think this is for a code example in the documentation. This should be code that is easy to read, "copy and pasteble" and with code styling that is closest to default javascript. I think trailing comas is not standard used in vanilla javascript. For the rest of the codebase, yes we should use trailing comas.

Reply:
There changes were done automatically by Prettier on save. Can roll them back if you want me to!